source/collections/Queue.ooc

```diff
@@ -89,4 +89,7 @@ VectorQueue: class <T> extends Queue<T> {
 		this _capacity = newCapacity
 	}
 	toString: func -> String { "Queue: count=" << this _count toString() >> " capacity=" & this _capacity toString() }
+	operator [] (index: Int) -> T {
+		this _backend[this _tail - index % this _capacity]
```

Comment:
Index 0 will return the last enqueued object. I considered the safe flag but failed to find usage examples so I didn't do it.